src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -213,13 +213,14 @@ internal ErrorCode Produce(string topic, byte[] val, int valOffset, int valLengt
                 // thread of the application calls poll() for a blocking produce() to ever unblock.
                 return LibRdKafka.producev(
                     handle,
```

Comment:
it would be nicer structurally to wrap `producev` in `LibRdKafka` and expose a method in which the VarTags don't need to be specified, however it's only a minor improvement structurally, everything's internal so it hardly matters and we may get a slight performance benefit if the extra level of indirection doesn't get optimized away.